regolith/tools.py

```diff
@@ -1497,21 +1497,21 @@ def collect_appts(ppl_coll, filter_key=None, filter_value=None, begin_date=None,
                 appts[-1].update({'person': p.get('_id')})
     return appts
 
-def get_grant_amount(grant, ppl_coll, begin_date=None, end_date=None):
+def grant_burn(grant, appts, begin_date=None, end_date=None):
     """
-    Given a grant, a database of people appointed on assorted grants, and two dates marking the limits of an interval,
-    it returns a list of the net amount of the grant on each day of the interval.
+    Retrieves the total burn of a grantt over an interval of time by integrating over all appointments
+    made on the grant.
 
     Parameters
     ----------
     grant: dict
-        The person whose appointments need to be checked
-    ppl_coll: collection (list of dicts)
-        The people collection that contains the grant appointments
+        The grant_id or grant object whose burn needs to be retrieved
+    appts: collection (list of dicts)
+        The collection of appointments made on assorted grants
     begin_date: datetime, string, optional
-        The start date of the interval of time to check the grant amount for
+        The start date of the interval of time to retrieve the grant burn for
```

Comment:
needs more I think.  What is the default behavior?  If string, does it have to be ain a particular format?

Reply:
done!